cmd/common/snapshot/process.go

```diff
@@ -23,6 +26,8 @@ import (
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/snapshot/process/types"
 )
 
+const OutputModeTree = "tree"
```

Comment:
nit: Maybe it is more accurate

```suggestion
const OutputModeProcessTree = "tree"
```